.github/workflows/release.yml

```diff
@@ -0,0 +1,61 @@
+# This GitHub action can publish assets for release when a tag is created.
+# Currently its setup to run on any tag that matches the pattern "v*" (ie. v0.1.0).
+#
+# This uses an action (paultyng/ghaction-import-gpg) that assumes you set your
+# private key in the `GPG_PRIVATE_KEY` secret and passphrase in the `PASSPHRASE`
+# secret. If you would rather own your own GPG handling, please fork this action
+# or use an alternative one for key handling.
+#
+# You will need to pass the `--batch` flag to `gpg` in your signing step
+# in `goreleaser` to indicate this is being used in a non-interactive mode.
+#
+name: release
+on:
+  push:
+    tags:
+      - 'v*'
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Go 1.13
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.13.x
```

Comment:
I'd rather use the same version for test and for [release](https://github.com/claranet/terraform-provider-zabbix/pull/19/files#diff-56c30d9224897b08c28075052d4595c5R45).

Why not go 1.15.2?

Reply:
Why not, I've just cut&paste [the boilerplate](https://github.com/hashicorp/terraform-provider-scaffolding/blob/master/.goreleaser.yml) referenced in https://www.terraform.io/docs/registry/providers/publishing.html#publishing-to-the-registry

Reply:
fixed